molecule/Dockerfile-rhel-java8.j2

```diff
@@ -3,3 +3,55 @@ FROM {{ item.image }}
 RUN yum -y install java-8-openjdk \
       rsync \
       openssl
+      rsyslog \
+      libselinux-python \
+      openldap \
+      openldap-servers \
+      compat-openldap \
+      openldap-clients \
+      openldap-devel \
+      nss-pam-ldapd \
+      libselinux-python \
+      krb5-libs \
+      krb5-server \
+      krb5-workstation \
+      unzip
+
+{% set DEFAULT_PACKAGE_VER = lookup('pipe', "awk '/confluent_package_version:/ {print $2}' $MOLECULE_PROJECT_DIRECTORY/roles/variables/defaults/main.yml" ) %}
+{% set PACKAGE_VER = lookup('env', 'VERSION') | default(DEFAULT_PACKAGE_VER, true) %}
+{% set REPO_VER = PACKAGE_VER | regex_replace('^([0-9])\\.([0-9]*).*', '\\1.\\2') %}
+{% set COMMON_REPO_URL = lookup('env', 'COMMON_REPO_URL') | default('https://packages.confluent.io', true) %}
+{% set CLIENT_REPO_URL = lookup('env', 'CLIENT_REPO_URL') | default('https://packages.confluent.io', true) %}
+
+RUN echo $'[Confluent.clients]\n\
+name=Confluent repository (clients)\n\
+baseurl={{ CLIENT_REPO_URL }}/clients/rpm/centos/\$releasever/\$basearch\n\
+gpgcheck=1\n\
+gpgkey={{ CLIENT_REPO_URL }}/clients/rpm/archive.key\n\
+enabled=1\n\
+\n\
+[Confluent]\n\
+name=Confluent repository\n\
+baseurl={{ COMMON_REPO_URL }}/rpm/{{ REPO_VER }}\n\
+gpgcheck=1\n\
+gpgkey={{ COMMON_REPO_URL }}/rpm/{{ REPO_VER }}/archive.key\n\
+enabled=1' >> /etc/yum.repos.d/confluent.repo
+
+RUN yum clean all && \
+    yum -y install confluent-common \
+      confluent-hub-client \
+      confluent-rebalancer \
+      confluent-rest-utils \
+      confluent-metadata-service \
+      confluent-server \
+      confluent-security \
+      confluent-kafka-connect-replicator \
+      confluent-kafka-rest \
+      confluent-ksqldb \
+      confluent-schema-registry \
+      confluent-control-center-fe \
+      confluent-control-center \
+      confluent-schema-registry-plugins \
+      confluent-ce-kafka-https-server \
+      confluent-ce-kafka-rest \
+      confluent-hub-client
```

Comment:
Can lead to lint error

Reply:
@nsharma-git sorry, what's the issue here?